### ISSUE TITLE
JSX Support

### ISSUE THREAD

**Issue by jessegood** [frankwallis/plugin-typescript#40]
I tried passing `"jsx": "react"` under `typescriptOptions` but it seems to have no effect.
Here is what my `config.js` looks like:
````
System.config({
  baseURL: "/",
  defaultJSExtensions: true,
  transpiler: "traceur",
  typescriptOptions: {
    "jsx": "react", <---- Doesn't seem to do anything
    // Omitting other stuff
  },
  paths: {
    "*": "src/*.tsx",
    "github:*": "jspm_packages/github/*",
    "npm:*": "jspm_packages/npm/*",
    "app": "src"
  },
  "packages": {
    "app": {
      "main": "main",
      "defaultExtension": "tsx",
      "meta": {
        "*.tsx": {
          "loader": "tsx"
        }
      }
    }
  },
  map: {
    "react": "npm:react@0.13.3",
    "traceur": "github:jmcriffey/bower-traceur@0.0.91",
    "traceur-runtime": "github:jmcriffey/bower-traceur-runtime@0.0.91",
    "tsx": "github:frankwallis/plugin-typescript@2.0.7",
````

Compiles fine as long as I don't use JSX syntax in the `.tsx` file.
Is it possible to pass `"jsx": "react"` to the compiler?

**Comment by frankwallis** [comment]
Thanks for raising this - it is fixed in 2.0.8 and I have added a working example.

**Comment by jessegood** [comment]
Thanks, it works!
